rlp/build.gradle

```diff
@@ -1,2 +1,6 @@
 
-description 'Ethereum Recursive Length Prefix (RLP) encoding for serializing objects'
\ No newline at end of file
+description 'Ethereum Recursive Length Prefix (RLP) encoding for serializing objects'
+
+dependencies {
+    compile project(':utils')
```

Comment:
Where is this dependency being used?

Reply:
this dependency provides `Numeric.hexStringToByteArray()`, test class needs this

Reply:
Shouldn't that be a test dependency then?

Reply:
@ZhiyuHelloWorld - did you see the above comment?

Reply:
just changed to test dependency